programs/bpf_loader/src/syscalls/cpi.rs

```diff
@@ -733,41 +739,89 @@ where
     Ok(accounts)
 }
 
+#[cfg(test)]
+static_assertions::const_assert_eq!(ACCOUNT_META_SIZE, 34);
```

Comment:
Looks like this belongs in a sepeerate pr

Reply:
Removed